.travis.yml

```diff
@@ -2,7 +2,8 @@ language: ruby
 sudo: false
 bundler_args: --jobs 4 --retry 2 --without packaging documentation
 before_install:
-  - gem update --system && gem install bundler --no-document
+  - yes | gem update --system 
+  - gem install bundler --no-document
```

Comment:
Is this command still necessary? As far as I know gem update --system / rubygems-update will provide the latest bundler.

Reply:
@dhoppe good catch. It seems like the versions installed using rvm and the ruby docker images come with a bundler gem so `gem update --system` updates to the latest bundler and `gem install bundler --no-document` is not needed.